### PR TITLE
[Developer] Add basic file format support for lexical model packages to Developer

### PR DESCRIPTION
**Pull request by mcdurdin**
This PR lets Keyman Developer load, save, edit and compile lexical model packages.

Does not support lexical models. That will be a separate PR.

### CODE REVIEW

**Comment by keyman-server** [comment]
[OSK renders for Windows](https://build.palaso.org/repository/download/KeymanDesktop_TestPrRenderOnScreenKeyboards/159992:id/kvk/index.html)

**Comment by keyman-server** [comment]
[OSK renders for Windows](https://build.palaso.org/repository/download/KeymanDesktop_TestPrRenderOnScreenKeyboards/160005:id/kvk/index.html)

**Comment by mcdurdin** [comment]
Force push because I branched from `master` instead of on `beta`  ... like a pro.

File: windows/src/version.txt [hunk 1]
```diff
@@ -1 +1 @@
-PRODUCTVERSION 11,0,1500,0
+PRODUCTVERSION 12,0,2000,0
```
Comment (mcdurdin) [inline windows/src/version.txt]: This was missed in the 12.0 changeover originally. Not great... Need to automate getting this from resources/version.md.

**Comment by keyman-server** [comment]
[OSK renders for Windows](https://build.palaso.org/repository/download/KeymanDesktop_TestPrRenderOnScreenKeyboards/160016:id/kvk/index.html)

File: windows/src/developer/TIKE/Tike.dpr [hunk 2]
```diff
@@ -301,8 +303,8 @@ begin
       if TikeActive then Exit;
       InitClasses;
       Application.CreateForm(TmodWebHttpServer, modWebHttpServer);
-      Application.CreateForm(TfrmKeymanDeveloper, frmKeymanDeveloper);
-      ShowStartup;
+  Application.CreateForm(TfrmKeymanDeveloper, frmKeymanDeveloper);
```
Comment (eddieantonio) [inline windows/src/developer/TIKE/Tike.dpr]: Was the indentation supposed to disappear in this file? Does it even matter?

Comment (mcdurdin) [reply windows/src/developer/TIKE/Tike.dpr]: Delphi automatically updates that file and flubs the indentation when it does. It doesn't matter; if I was a complete purist I'd go back and tidy it up again 😁

**Comment by keyman-server** [comment]
[OSK renders for Windows](https://build.palaso.org/repository/download/KeymanDesktop_TestPrRenderOnScreenKeyboards/160094:id/kvk/index.html)

**Pull request merged**